packages/server/src/definitions/automations.ts

```diff
@@ -13,19 +13,17 @@ export interface TriggerOutput {
   timestamp?: number
 }
 
-export interface AutomationContext extends AutomationResults {
+export interface AutomationContext {
+  trigger: any
```

Comment:
Should this not be an AutomationTrigger?

Reply:
It's the output of an `AutomationTriggerResult`. I've created a new type for this that aliases `Record<string, any>`, because I don't think we have better typing that than yet.